care/facility/api/serializers/facility.py

```diff
@@ -95,7 +98,10 @@ class FacilitySerializer(FacilityBasicInfoSerializer):
     # }
     read_cover_image_url = serializers.URLField(read_only=True)
     # location = PointField(required=False)
-    features = serializers.MultipleChoiceField(choices=FEATURE_CHOICES)
+    features = serializers.ListField(
+        child=serializers.ChoiceField(choices=FEATURE_CHOICES),
+        required=False,
+    )
```

Comment:
can you check if the validations are working 
- should not allow values not present in conices
- should not allow duplicates